### DISCUSSION TITLE
Remove/delete/purge

### DISCUSSION THREAD

**Discussion by dantveita** [ObsidianPublisher/obsidian-github-publisher#107] [Q&A — Plugin]
Hi,

Im a little confused about how to remove deleted or "share: false" notes. As I understood it, it should be possible to remove notes from github like that. Can anyone explain to me how that works?

**Comment by Mara-Li** [comment]
Did you activate the autoclean option in the settings ?

**Reply by dantveita** [reply]
Yes, without success. Theres a notification that says it has started purging, but there is no notification telling me its done, though.

**Reply by dantveita** [reply]
I just noticed: Everytime I restart obsidian, Autoclean option is turned off 🤔

**Reply by Mara-Li** [reply]
What's your settings ? Autoclean is always disabled on fixed folder

**Comment by gilipsy** [comment]
<img width="748" alt="Снимок экрана 2023-07-09 в 1 27 29 AM" src="https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/133940693/e3378c35-e879-4024-9dee-4ac170721ea8">
With these settings, everything works and deletes.

**Comment by longnetpro** [comment]
Auto clean up option cannot be enabled whatever the folder type is. Please fix it.

The version of OB is 1.3.7 and the plugin version is 6.4.0

**Reply by Mara-Li** [reply]
It will be 15$.

**Reply by longnetpro** [reply]
Thanks. I am using Obsidian Git now. More familiar with git itself.
